pages/page_elements.js

```diff
@@ -0,0 +1,50 @@
+class PageElements {
+  constructor(page) {
+    this.page = page;
+  }
+
+  async getProductsButton() {
+    return this.page.getByText("Products");
```

Comment:
The `getByText` method is sync. Please remove async for the `getProductsButton()` method